charts/backstage/values.yaml

```diff
@@ -245,6 +248,14 @@ upstream:
       enabled: false
       path: /metrics
       port: http-metrics
+  # -- Catalog index configuration for automatic plugin discovery.
+  # The `install-dynamic-plugins.py` script pulls this image if the `CATALOG_INDEX_IMAGE` environment variable is set.
+  # The `dynamic-plugins.default.yaml` file will be extracted and written to `dynamic-plugins-root` volume mount.
+  catalog-index:
```

Comment:
Hmm... A bit confusing to see this under `upstream` IMO.. But there might be an issue if moved to the root level, as the value is being consumed in the `backstage` sub-chart.
So let's move this back to the `global` scope (but not under `global.dynamic`), i.e.: `global.catalogIndex`